content/source/docs/cloud/workspaces/index.html.md

```diff
@@ -7,18 +7,43 @@ page_title: "Workspaces - Terraform Cloud"
 
 Workspaces are how Terraform Cloud organizes infrastructure.
 
-A workspace consists of:
+## Workspaces are Collections of Infrastructure
 
-- A Terraform configuration (usually retrieved from a VCS repo, but sometimes uploaded directly).
-- Values for variables used by the configuration.
-- Persistent stored state for the resources the configuration manages.
-- Historical state and run logs.
+Working with Terraform involves _managing collections of infrastructure resources,_ and most organizations manage many different collections.
+
+When run locally, Terraform manages each collection of infrastructure with a _persistent working directory,_ which contains a configuration, state data, and variables. Terraform CLI uses content from the directory it runs in; by using separate directories, it's easy to organize infrastructure resources into meaningful groups.
```

Comment:
Hmm, yeah.

Reply:
> Since Terraform CLI uses content from the directory it runs in, you can organize infrastructure resources into meaningful groups by keeping their configurations in separate directories.